src/main/java/one/util/streamex/MoreCollectors.java

```diff
@@ -168,6 +169,188 @@ private MoreCollectors() {
         }, Function.identity(), set -> set.size() == size, UNORDERED_ID_CHARACTERISTICS);
     }
 
+    /**
+     * Returns a {@code Collector} that accumulates elements into a {@code Map}
+     * whose keys and values are taken from {@code Map.Entry}.
+     *
+     * @param  <K> the {@link Comparable} type of then map keys
+     * @param  <V> the type of the map values
+     *
+     * @return {@code Collector} which collects elements into a {@code Map}
+     * whose keys and values are taken from {@code Map.Entry}
+     * @throws IllegalStateException if this stream contains duplicate keys
+     *         (according to {@link Object#equals(Object)}).
+     *
+     * @see #entriesToMap(BinaryOperator)
+     * @see #entriesToMap(Function, BinaryOperator)
+     * @see Collectors#toMap(Function, Function)
+     * @since 0.7.3
+     */
+    public static <K, V> Collector<Entry<? extends K, ? extends V>, ?, Map<K, V>> entriesToMap() {
+        return Collectors.toMap(Entry::getKey, Entry::getValue);
```

Comment:
I suggest delegating to `entriesToCustomMap(HashSet::new)`. This is shorter, and users will benefit from a more detailed exception message in case of duplicated keys on Java 8.